core/src/test/scala/com/pingcap/tispark/datasource/BasicSQLSuite.scala

```diff
@@ -32,11 +32,12 @@ class BasicSQLSuite extends BaseBatchWriteWithoutDropTableTest("test_datasource_
     jdbcUpdate(s"insert into $dbtable values(null, 'Hello'), (2, 'TiDB')")
   }
 
-  test("Test Select") {
+  // temporarily not supported
+  ignore("Test Select") {
     testSelectSQL(Seq(row1, row2))
   }
 
-  test("Test Insert Into") {
+  ignore("Test Insert Into") {
```

Comment:
Please add some comments?
- why ignore
- Should it be supported in the future? when and how, add TODO